scripts/prepare

```diff
@@ -3,5 +3,6 @@
 patch-package
 
 npm run dist
+npm run build
```

Comment:
This script is missing in package.json and when I execute npm run prepare, it throws an npm error.

Reply:
it's a leftover, I will remove it, thanks.